gcc/config/riscv/riscv.c

```diff
@@ -1809,6 +1826,17 @@ riscv_output_move (rtx dest, rtx src)
     }
   gcc_unreachable ();
 }
+
+
```

Comment:
Most functions are separated by a single blank line, and you have two here, but this is a minor detail.